src/SDK/Resource/Detectors/Host.php

```diff
@@ -66,10 +72,15 @@ private function getLinuxId(): ?string
     private function getBsdId(): ?string
     {
         $file = $this->dir . self::PATH_ETC_HOSTID;
-        if (is_file($file) && is_readable($file)) {
-            $contents = file_get_contents($file);
 
-            return $contents !== false ? trim($contents) : null;
+        try {
+            if (is_file($file) && is_readable($file)) {
+                $contents = file_get_contents($file);
+
+                return $contents !== false ? trim($contents) : null;
```

Comment:
Should not return here if `file_get_contents()` returns false, we still want to attempt the fallback.

Reply:
updated to disable the error handler